pkg/asset/installconfig/powervs/regions.go

```diff
@@ -0,0 +1,43 @@
+package powervs
+
+import (
+	"github.com/openshift/installer/pkg/rhcos"
+)
+
+func knownRegions() map[string]string {
+
+	regions := make(map[string]string)
+
+	for _, region := range rhcos.PowerVSRegions {
+		regions[region["name"]] = region["description"]
+	}
+	return regions
+}
+
+// IsKnownRegion return true is a specified region is Known to the installer.
+// A known region is subset of AWS regions and the regions where RHEL CoreOS images are published.
```

Comment:
oops. missed the `AWS` here. we can just make a note of it and have some cleanup for when we take out any TODO comments, etc.